x-pack/plugins/apm/public/components/routing/app_root/index.tsx

```diff
@@ -132,7 +132,10 @@ function MountApmHeaderActionMenu() {
 }
 
 export function ApmThemeProvider({ children }: { children: React.ReactNode }) {
-  const [darkMode] = useUiSetting$<boolean>('theme:darkMode');
+  const {
+    services: { theme },
+  } = useKibana();
+  const darkMode = useMemo(() => theme?.getTheme().darkMode ?? false, [theme]);
```

Comment:
Is it really hard to get the theme that needs to be wrapped around `useMemo`?

Reply:
No, I can revert the memo usage if you prefer, just tell me!